modules/lnd/lnd_wrapper/wrapper.go

```diff
@@ -162,6 +162,49 @@ func LndParseP2pLightningMessage(data *C.char, length C.int) *C.char {
 	case 19:
 		sb.WriteString("MSG_TYPE=pong;IGNORED=")
 		sb.WriteString(fmt.Sprintf("%d", len(message.(*lnwire.Pong).PongBytes)))
+	case 35:
+		_, err := message.(*lnwire.FundingSigned).CommitSig.ToSignature()
```

Comment:
Let's cast just once at the beginning here.

```suggestion
	case 35:
		fs := message.(*lnwire.FundingSigned)
		_, err := fs.CommitSig.ToSignature()
```

Reply:
Updated!